### PR TITLE
PTR-5

### PR DESCRIPTION
**Pull request by artem-dehnych**

### CODE REVIEW

**Review by mitssev — APPROVED** [review]

File: .github/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: build and test
+on:
+  push:
+    branches: [ "main" ]
```
Comment (Arrrmy21) [inline .github/ci.yaml:4]: Does it mean that action won't trigger on any other branch except 'main'?

File: .github/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: Build with Maven
+        run: mvn clean package -DskipTests
```
Comment (Arrrmy21) [inline .github/ci.yaml:20]: Are we going to skip tests while building or it's a copy-paste mistake?

File: .github/ci.yaml (hunk 1 above)
Comment (artem-dehnych) [reply .github/ci.yaml:4]: yes

File: .github/ci.yaml (hunk 2 above)
Comment (artem-dehnych) [reply .github/ci.yaml:20]: yes, This build phase.

**Review by Arrrmy21 — APPROVED** [review]

**Pull request merged**